jenkins/opensearch/benchmark-pull-request.jenkinsfile

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@6.6.0', retriever: modernSCM([
```

Comment:
Should we use the same version as the one in benchmark-test job?
This 6.6.0 is only adding changes for gradle check.

Thanks.

Reply:
Both have the same code for benchmark tests, this should have no effect on the functionality.